WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -437,9 +437,10 @@ def execute( self ):
     if self.pp.localConfigFile:
       cfg.append( self.pp.localConfigFile )  # this file is as input
 
-    checkCmd = "dirac-wms-get-wn-parameters %s" % ' '.join( cfg )
+    checkCmd = 'dirac-wms-get-wn-parameters -S %s -N %s -Q %s %s' % ( self.pp.site, self.pp.ceName, self.pp.queueName,
+                                                                       " ".join( self.cfg ) )
```

Comment:
This is now using "self.cfg" instead of "cfg". "cfg" is still being filled in the previous lines, so is this change correct?